x-pack/plugins/alerts/server/invalidate_pending_api_keys/mark_api_key_for_invalidation.test.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { loggingSystemMock, savedObjectsClientMock } from '../../../../../src/core/server/mocks';
+import { markApiKeyForInvalidation } from './mark_api_key_for_invalidation';
+
+describe('markApiKeyForInvalidation', () => {
+  test('should call savedObjectsClient create with the proper params', async () => {
```

Comment:
Looks like we are missing tests for failure scenarios. Do you think it's worthwhile to exercise those code paths here?